source/community.html.erb

```diff
@@ -2,12 +2,75 @@
+<a href="https://github.com/hanami" target="_blank">github.com/hanami</a>
+
+<h2 class="page-header">Chat Room</h2>
+<p>We have an official chat room on Gitter. If you need a quick question answered, this is the best place to do it. Also, if you are working on a project and you need any kind of advice, please feel free to ask. Someone will be there to answer most of the times. We are always happy to help.</p>
+<a href="https://gitter.im/hanami/chat" target="_blank">hanami/chat</a>
+
+<h2 class="page-header">Blog</h2>
+<p>On the official Hanami blog, you can find the detailed announcements for new releases, some interesting use cases and things the team writes around the code they usually write.</p>
+<a href="/blog">Hanami Blog</a>
+
+<h2 class="page-header">Discussion Forum</h2>
+<p>The Hanami forum is a good place for general discussions. Feel free to ask a question, share your project with others, ask for guidance and best practices, offer your help contributing to a project, or anything else.</p>
+<a href="https://discourse.hanamirb.org" target="_blank">discourse.hanamirb.org</a>
+
+<h2 class="page-header">Twitter</h2>
+<p>We regularly tweet general announcements on releases and events where Hanami is represented in any way. We also retweet interesting tweets from the Hanami and Ruby community. Please mention <code>@hanamirb</code> or use <code>#hanamirb</code> if you are tweeting about Hanami.</p>
+<a href="https://twitter.com/hanamirb" target="_blank">@hanamirb</a>, <a href="https://twitter.com/search?q=hanamirb" target="_blank">#hanamirb</a>
+
+<h2 class="page-header">Reddit</h2>
+<p>On our reddit page, we link blog posts about new releases, have links to useful Hanami learning material, and have interesting discussions with the Hanami community.</p>
+<a href="https://www.reddit.com/r/hanamirb/" target="_blank">r/hanamirb</a>
+
+<h2 class="page-header">StackOverflow</h2>
+<p>If you have any questions about Hanami, you can always ask on StackOverflow. Make sure you use the Hanami tag. You can also always help others solving their problems. You will make others happy, as you are when you get help.</p>
+<a href="https://stackoverflow.com/questions/tagged/hanami" target="_blank">hanami tag</a>
+
+<h2 class="page-header">Awesome Hanami</h2>
+<p>The awesome-hanami Github repository lists Hanami gems, libraries and projects that use Hanami. Also, there are some useful links to know more about Hanami and its users.</p>
+<a href="http://awesome-hanami.org" target="_blank">awesome-hanami.org</a>
+
+<h2 class="page-header">Mailing List</h2>
+<p>To stay updated with the latest releases, to receive code examples, implementation details and announcements, please consider to subscribe to the <a href="/mailing-list">Hanami mailing list</a>.</p>
+
+<h2 class="page-header">Conference Talks about Hanami</h2>
```

Comment:
IMHO this section could be in a separate section. What do you think about a separate section for `Learning`?